### DISCUSSION TITLE
default value binding will not be refreshed

### DISCUSSION THREAD

**Discussion by samg666** [Budibase/budibase#3132] [Help]
Hello,

I have bind value on the text field Default Value and Placeholder, if the binding value was changed, the Placeholder binded value will be synced, but the Default Value always keep the first binded value.  It seems that the Default Value binding only bind on the first time? Is it possible to make the Default Value to be synced with the binded value all the time only if manually changed?

![iShot2021-10-25 12 39 31](https://user-images.githubusercontent.com/29649651/138635554-f71ef62b-05e5-448b-8c8d-59e0c18b9940.png)

**Comment by mjashanks** [comment]
Hi Sam.

It's not possible to do as you described. The default value cannot be used in this way. What you could do is add an "Override default" or "Use Other" checkbox, as shown in the video below.

My binding in the `Paragraph`, which shows the resulting email, looks like 
```
{{ #if New Form.Fields.overRideemail }}{{ New Form.Fields.Email }}{{ else }} {{ New Form.Fields.Person }}  {{/if}}
```

https://user-images.githubusercontent.com/3524181/138681482-6907dbf7-4ed1-4a8f-b5a3-24e693d9ca9f.mp4

**Reply by samg666** [reply]
Hi Michael,

@mjashanks this is not work in my case. The text field default value just bind on the page load at the first time, and it will not changed even the binded value actually changed, the only way to make the default value change is to refresh the page. I don't understand why design like this? Is there any chance that you can make the default value change when the binding value changed(just like the binding on paragraph)?

My use case is, I have a dropdown list named 'staffs', when the selected staff changed, then my data provider will pull this staff detail information, then I will bind/pre-fill on other text fields.

https://user-images.githubusercontent.com/29649651/142593522-82537882-d259-4a34-9dff-f284c6bb0fb2.mov

**Reply by dougwettlaufer** [reply]
I'm running into a similar issue, have there been any updates for this? I've also tried to use the "Update setting" condition to change the default value. The condition seems to be firing based on some console.log() I dropped in but the value in the text box isn't changing.

**Reply by dougwettlaufer** [reply]
My workaround ended up being to add another data provider and form inside of the previous form. So the dropdown is populated by data provider A and then my other field is populated by data provider B which is filtered based on the contents of the dropdown. It's not ideal since I now have multiple data providers of the same type but it works 🤷‍♂️
